# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/public/friend/UsageStatistics.h
# Commit: 71d2bf08d2da
# Full Hash: 71d2bf08d2da8edd6e390cf38eee723d8d4733d8
# Author: Denis Palmeiro <dpalmeiro@mozilla.com>
# Date: 2025-01-21 21:28:51
# Description:
#   Bug 1942415: Revert changes for GC_GLEAN_SLOW_PHASE and GC_GLEAN_SLOW_TASK probes.  r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D234808
# ==============================================================================

diff -r 19e1106de68c -r 71d2bf08d2da js/public/friend/UsageStatistics.h
--- a/js/public/friend/UsageStatistics.h	Tue Jan 21 16:08:01 2025 +0200
+++ b/js/public/friend/UsageStatistics.h	Tue Jan 21 14:28:44 2025 +0000
@@ -75,10 +75,7 @@
  * However, unlike the legacy list, each glean metric must be manually added
  * to the switch statement in AccumulateTelemetryCallback().
  */
-#define FOR_EACH_JS_GLEAN_METRIC(_)    \
-  _(ION_COMPILE_TIME, TimeDuration_US) \
-  _(GC_GLEAN_SLOW_PHASE, Enumeration)  \
-  _(GC_GLEAN_SLOW_TASK, Enumeration)
+#define FOR_EACH_JS_GLEAN_METRIC(_) _(ION_COMPILE_TIME, TimeDuration_US)
 
 #define FOR_EACH_JS_METRIC(_)  \
   FOR_EACH_JS_LEGACY_METRIC(_) \